fml/platform/darwin/cf_utils.h

```diff
@@ -36,14 +51,20 @@ class CFRef {
     instance_ = instance;
   }
 
+  [[nodiscard]] T Release() {
```

Comment:
Are we on C++ 17 everywhere we have to compile?  Particularly google3?

Reply:
Yes.